tools/lint/lint.xml

```diff
@@ -117,4 +117,8 @@
 
     <!-- Compose -->
     <issue id="UnnecessaryComposedModifier" severity="error" />
+
+    <!-- There seems to be an issue with this check, it flags lots of false positives. -->
+    <!-- TODO: check again in the near future. -->
```

Comment:
Maybe add a link to https://issuetracker.google.com/issues/349411310 ?

Reply:
https://github.com/element-hq/element-x-android/pull/3594/commits/e49a83802a5a3f689b856d837b15ee786c536e03